pkg/controllers/metrics/pod/controller.go

```diff
@@ -71,18 +72,47 @@ var (
 			Objectives: metrics.SummaryObjectives(),
 		},
 	)
+	podBoundDurationSeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "karpenter",
+			Subsystem: metrics.PodSubsystem,
+			Name:      "bound_duration_seconds",
+			Help:      "The time from pod creation until the pod is bound.",
+			Buckets:   metrics.DurationBuckets(),
```

Comment:
Our `pod_startup_duration_seconds` doesn't have any additional labels -- it's also not a histogram like it should be. At this point, we already marked it as stable so we can't change it from a summary metric. We can consider marking it as deprecated and replace it with a histogram metric called `karpenter_pods_start_duration_seconds`.

The other option: According to [the docs](https://prometheus.io/docs/concepts/metric_types/), the only difference between the summary metric and the histogram metric is the addition of the `_bucket` metric vs a metric on the basename that has the label (`quantile`).  That means that we could have them existing simultaneously as a histogram and summary metric -- we should look into if this is possible to avoid the breaking change/rename